dotnet/csharp.bzl

```diff
@@ -407,6 +417,104 @@ def csharp_configure():
       actual = "@local_config_csharp//:csc_bin",
   )
 
+def _nuget_dep_impl(ctx):
+  inputs = set(ctx.files.srcs)
+  return struct(
+    name = ctx.label.name,
+    target_type = ctx.attr._target_type,
+    out = inputs,
+    dlls = inputs,
+    transitive_dlls = set([]),
+  )
+
+nuget_dep = rule(
```

Comment:
It certainly might. I haven't looked at the java_import rule. It's basically just a way to reference other dlls. perhaps it should just be called dll_import or dotnet_import?

It's also possible that some refactoring of the csharp_* rules could make this thin wrapper unnecessary and we could just use filegroups. This was the minimal I could do to make it work.